src/app/index.ts

```diff
@@ -21,6 +21,7 @@ const excelCustomFunctions = `excel-functions`;
 const manifest = 'manifest';
 const typescript = `Typescript`;
 const javascript = `Javascript`;
+let language = undefined;
```

Comment:
I think you would get a lint error here since you shouldn't need to initialize a variable to `undefined`. `Let language;` is sufficient.

Is tslint being used for generator-office?